components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/OIDCSessionConstants.java

```diff
@@ -65,6 +65,22 @@ public static class OIDCEndpoints {
         public static final String OIDC_LOGOUT_ENDPOINT = "/oidc/logout";
     }
 
+    /**
+     * Contains the constants related to OIDC logout request sender.
+     */
+    public static class OIDCLogoutRequestConstants {
+
+        public static final String POOL_SIZE = "OAuth.OpenIDConnect.LogoutRequestSender.PoolSize";
+        public static final String KEEP_ALIVE_TIME = "OAuth.OpenIDConnect.LogoutRequestSender.KeepAliveTime";
+        public static final String HTTP_CONNECT_TIMEOUT = "OAuth.OpenIDConnect.LogoutRequestSender.HttpConnectTimeout";
+        public static final String HTTP_SOCKET_TIMEOUT = "OAuth.OpenIDConnect.LogoutRequestSender.HttpSocketTimeout";
+
+        public static final String DEFAULT_POOL_SIZE = "2";
```

Comment:
What is the default value that we are going to put in the file? I think better to have that in here as well. My opinion is to have something like 10 or 20.

Reply:
Addressed via 696835c1e65a4a4e34981c2dfe9bc7c9ccd23e71